module/applications/actor/base-sheet.mjs

```diff
@@ -38,12 +38,28 @@ export default class ActorSheetEd extends ActorSheet {
     return expandObject( enrichment );
   }
 
+  async _enableHTMLEnrichmentItem( itemC ) {
```

Comment:
shouldn't these functions rather be called something like `enrichHTML` and something like `enrichtItemHTML`?

also, this is a fringe case where you have to think about whether it is part of the sheet or document class. because what the function actually does, is modifying the document's data. but it's only relevant for displaying the sheet.
 I would probably factor it out to the respecitve document classes, so you can just call `actor.enrichHTML` and `item.enrichtHTML`, enriching all necessary fields.

Reply:
those functions are called enableHTMLEnrichment everywhere, I would rather leave it like this to better find it if anything causes an issue.

I don't get it out of the getData

Reply:
oki doki